src/Modal/Modal.js

```diff
@@ -46,24 +48,32 @@ const Modal = ({
 
   return (
     <Fragment>
-      {!offModal && renderModal}
+      {modalStatus && renderModal}
+      {
+        toggleButton
+        && <Button modifiers="outline" onClick={() => setModalStatus(!modalStatus)}>{toggleButton}</Button>
+      }
     </Fragment>
   );
 };
 
 Modal.propTypes = {
-  simple: PropTypes.bool,
-  alert: PropTypes.bool,
   title: PropTypes.string,
+  toggleButton: PropTypes.string,
   actions: PropTypes.array,
```

Comment:
I would try to have more detailed props, in this case for example:
`
actions:PropTypes.arrayOf(
  PropTypes.arrayOf(
    PropTypes.string,
    PropTypes.func
  )
)
`

This will give us more info if we use the component and we don't respect the format props